### DISCUSSION TITLE
Pie chart

### DISCUSSION THREAD

**Discussion by SukhorukovS** [TanStack/react-charts#283] [Ideas]
Do you have a pie chart in the library or maybe planning to add it?

**Comment by paulgrieselhuber** [comment]
Same question

**Comment by joshuabaker** [comment]
> [purposefully does not have support for pie charts, radar charts, or other circular nonsense.](https://www.businessinsider.com/pie-charts-are-the-worst-2013-6)

**Reply by joshuabaker** [reply]
From https://react-charts.tanstack.com/

**Reply by bosswissam** [reply]
Thank you for sharing. Really disappointing. It's one thing for a statistician to not want to use pie charts to visualize some data, it's another for a charting library to not give it's users the option to use them. Very limiting and random decision by the maintainers, and they BI article linked focuses on use cases you'd never use a pie chart for anyway. Really random and confusing.
